routes/user/user.js

```diff
@@ -286,6 +286,14 @@ exports.createUser = function(req, res, next) {
                         res.send(user);
                     }
                 });
+            common.sendEmail("harvestapi@rada.gov.jm", "New HarvestAPI User", "A new user with email address " + user.us_email_address + " has signed up to use the API.", function (error, info) {
```

Comment:
The `harvestapi@rada.gov.jm` email address should be configurable as an environment variable - not hardcoded.